narwhal/benchmark/benchmark/logs.py

```diff
@@ -158,12 +198,17 @@ def _parse_workers(self, log):
         tmp = findall(r'Batch ([^ ]+) contains sample tx (\d+)', log)
         samples = {int(s): d for d, s in tmp}
 
+        tmp = findall(
```

Comment:
Does this still work with logging at debug level?

Reply:
Yeah the benchmarks run with debug logs on